source/guide_nextcloud.rst

```diff
@@ -9,7 +9,7 @@
 Nextcloud
 #########
 
-Nextcloud_ is an open source cloud solution written in PHP and distributed unter the AGPLv3 licence.
+Nextcloud_ is an open source cloud solution written in PHP and distributed unter the AGPLv3 licence. You can find further infromation on how to secure and optimise your setup at the offical `docs page <https://docs.nextcloud.com>`_.
```

Comment:
Good idea, but the link should probably be at the end of the article

Reply:
Ok. I will have a look at resizing the logo so this text can be moved to the end oft the tutorial.

Reply:
the logo overlapping the box below is a general problem, which shouldn't be solved on a per-guide basis, imho.